README.rst

```diff
@@ -49,6 +49,28 @@ Get involved!
 -  Read more about the project's motivation at `Introducing KA Lite, an offline version of Khan
    Academy <http://jamiealexandre.com/blog/2012/12/12/ka-lite-offline-khan-academy/>`__.
 
+Roadmap
+-------
+
+During 2017, Learning Equality is launching the successor of KA Lite. It's
+called `Kolibri <http://github.com/learningequality/kolibri>`__ and will have
+all the features of KA Lite, but also be a platform for many other offline
+educational sources.
+
+Because of the reported popularity of KA Lite, we are continuing our
+efforts to support deployments by providing fixes problems that may occur and
+thus directly effect current usage. For instance issues related to new
+browsers, operating systems etc. We are also still optimizing regarding
+performance issues.
+
+If you are creating a new deployment at this very moment, feel assured that
+KA Lite is still alive and will be maintained for the rest of 2017.
+
+In case you need KA Lite to continue to grow its features, you should join the
```

Comment:
Perhaps "In the meantime, if you need new features in KA Lite, we welcome you to join the community and contribute."